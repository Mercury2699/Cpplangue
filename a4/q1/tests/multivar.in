x y + z * alpha + beta - done
print
eval

set x 1 set y 2 set z 3
set alpha 8 set beta 9

print
eval

set x 5
print eval

unset alpha
print
eval

unset x
print
eval
