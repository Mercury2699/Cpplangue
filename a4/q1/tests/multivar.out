((((x + y) * z) + alpha) - beta)
((((x + y) * z) + alpha) - beta)
x has no value.
((((1 + 2) * 3) + 8) - 9)
8
((((5 + 2) * 3) + 8) - 9)
20
((((5 + 2) * 3) + alpha) - 9)
alpha has no value.
((((x + 2) * 3) + alpha) - 9)
x has no value.
